pdns/dnsdistdist/test-dnsdistasync.cc

```diff
@@ -84,7 +84,8 @@ BOOST_AUTO_TEST_CASE(test_Basic)
     struct timeval ttd;
     gettimeofday(&ttd, nullptr);
     // timeout in 100 ms
-    ttd.tv_usec += 100000;
+    const timeval sub{0, 100000};
+    timersub(&ttd, &sub, &ttd);
```

Comment:
A `+=` changed into a `timersub` here

Reply:
There are a few other instances of timeval computations left.

Reply:
:+1: